harness/determined/pytorch/_lr_scheduler.py

```diff
@@ -5,6 +5,10 @@
 
 import determined_common.check as check
 
+TorchLRScheduler = typing.Union[
```

Comment:
? expose this in `__init__` since user facing

Reply:
it's not really user facing though right? We just use this internally here to make mypy happy

Reply:
I think input types for user facing functions/classes (such as `LRScheduler`)  should be considered user facing.

Reply:
I personally don't really see the rationale. They're going to import it and do what with it? If you think it's important and it's something we've done elsewhere then I'll do it

Reply:
It seems like we do that for types that users will have to implement a method for (`TorchData`). I'll do it for this one anyways 👍

Reply:
`keras.InputData` is a similar thing although it's an expected output vs. arg. But yeah I do agree with you, we should just discuss this in #ml-ag rather than trying to figure out the standard in this thread.